scipy/optimize/_milp.py

```diff
@@ -0,0 +1,359 @@
+import warnings
+import numpy as np
+from scipy.sparse import csc_array, vstack
+from ._highs._highs_wrapper import _highs_wrapper  # type: ignore[import]
+from ._constraints import LinearConstraint, Bounds
+from ._optimize import OptimizeResult
+
+
+def _constraints_to_components(constraints):
+    # convert sequence of constraints to a single set of components A, b_l, b_u
```

Comment:
Is this better as a docstring instead of single comment?

Reply:
I guess it would be slightly more useful that way, yes.

Reply:
@tupui beat me to it